books/bookstore.rb

```diff
@@ -4,110 +4,154 @@
 
 log = File.open("#{Rails.root.to_s}/books/bookstore_#{Time.now.strftime("%Y.%m.%d-%H:%M")}.log", 'w')
 
+# Monitor logging in real-time w/ tail -f
+log.sync = true
+
 initial_time = Time.now
 
 
-puts 'Wiping books and book_requirements...'
+puts 'Wiping book_requirements...'
 
-Book.delete_all
-BookRequirement.delete_all
+semester_id = Semester.find_by(:season => 'Fall', :year => 2015).id
 
-departments = Nokogiri::Slop(RestClient.get('http://uvabookstores.com/uvatext/textbooks_xml.asp?control=campus&campus=77&term=92')).departments.department
-departments = [departments] if departments.class == Nokogiri::XML::Element
-departments.each do |xml_department|
-	mnemonic = xml_department['abrev']
-	department_id = xml_department['id']
+ActiveRecord::Base.connection.execute("DELETE br.* FROM book_requirements AS br JOIN sections AS s ON s.id=br.section_id WHERE s.semester_id=#{semester_id}")
```

Comment:
what was wrong with using orm? should use orm... as orm can do dependent destroy as well